src/components/ArticleList.jsx

```diff
@@ -1,23 +1,22 @@
 import { useState, useEffect } from 'react';
 import { fetchArticles } from "./api";
-import { useParams } from 'react-router-dom';
+import { useParams, Link } from 'react-router-dom';
 // import { UserContext } from '../contexts/UserContext'
 // import { useContext } from 'react'
 
-
 const ArticleList = () => {
 
+    console.log('--sanity check')
```

Comment:
Please remove console.logs before making the PR this should be production code standard